runtime/onert/backend/cpu/ops/TransposeLayer.cc

```diff
@@ -38,10 +39,22 @@ template <typename T> void TransposeLayer::transpose()
 {
   nnfw::cker::TransposeParams param;
   assert(_perm->num_dimensions() == 1);
-  param.perm_count = _perm->dimension(0);
-  for (auto i = 0; i < param.perm_count; i++)
+
+  param.perm_count = _input->num_dimensions();
+  if (_perm->dimension(0) == 0) // Regular Transpose
```

Comment:
When I first see this PR, I was curious about the definition of "regular transpose".
I googled the definition but it was hard to find. Could you explain a bit?

Reply:
It is not a officially term.
In linear algebra, the transpose of a matrix is an operator which flips a matrix over its diagonal. It switches the row and column indices of the matrix A by producing another matrix.
In neural networks, it is expanded in multiple dimensions with a vector specifying how the multiple dimensions will be permuted. If the vector is regularly set to (n-1...0), it is the same as expanded transpose of linear algebra and as Transpose operation of nnapi without perms. So I commented it as "Regular Transpose".

Reply:
I see. 
I am afraid that the term "Regular Transpose" may _not_ be meaningful to most people.
Why don't you remove `Regular Transpose` throughout the code?

Reply:
Okay I will change it to (n-1...0)